04-vector-raster-integration-advanced.Rmd

```diff
@@ -298,46 +322,49 @@ CHM_HARV_manualCrop <- crop(x = chm_HARV, y = new.extent)
 #plot extent boundary and newly cropped raster
 plot(aoiBoundary_HARV, 
      main = "Manually Cropped Raster\n NEON Harvard Forest Field Site")
-plot(new.extent, col="blue", lwd=4,add = TRUE)
+plot(new.extent, col="red", lwd=4,add = TRUE)
```

Comment:
changed to orange.  Originally changed away from blue as blue, green, purple, and black refer to specific layers/extents in this lesson.